src/relay/ir/type.cc

```diff
@@ -22,6 +22,14 @@ TensorType TensorTypeNode::Scalar(DataType dtype) {
   return TensorTypeNode::make({}, dtype);
 }
 
+IndexExpr TensorTypeNode::Size() const {
+  IndexExpr size = make_const(Int(64), 1);
```

Comment:
directly product from shape[0], when the shape.size() equals 0, return make_const(Int(32), 1);